docs/reference/release-notes/highlights.asciidoc

```diff
@@ -17,95 +17,13 @@ Other versions:
 | {ref-bare}/8.1/release-highlights.html[8.1]
 | {ref-bare}/8.0/release-highlights.html[8.0]
 
+// The notable-highlights tag marks entries that
+// should be featured in the Stack Installation and Upgrade Guide:
 // tag::notable-highlights[]
-
```

Comment:
I don't think that these release highlights should be removed. The published [highlights for 8.4](https://www.elastic.co/guide/en/elasticsearch/reference/current/release-highlights.html) have several inclusions, which should remain. I think that this entire section needs to stay.

Reply:
Agreed. The info for 8.4 should remain as-is.

Reply:
Restored the content.